std/process.d

```diff
@@ -878,12 +878,7 @@ private Pid spawnProcessImpl(scope const(char[])[] args,
 
     if (args.empty) throw new RangeError();
     const(char)[] name = args[0];
-    if (any!isDirSeparator(name))
-    {
-        if (!isExecutable(name))
-            throw new ProcessException(text("Not an executable file: ", name));
-    }
-    else
+    if (!any!isDirSeparator(name))
```

Comment:
With this new check, won't path be prepended to all checks? So if you try to run "/bin/sh", it would try "/usr/bin//bin/sh", etc.?

Perhaps the better check is, does it start with a directory separator?

Reply:
Hm, nevermind. If there is NO path separator, then it searches. I was misreading the check.

Reply:
Yes, same as what `execvp` etc. do. "The execlp(), execvp(), and execvpe() functions duplicate the actions of the shell in searching for an executable file **if the specified filename does not contain a slash (/) character**."